packages/ui/ui-theme/src/css/layout/main.css

```
@@ -80,9 +80,10 @@
     .dark & {
       @apply backdrop-blur-lg;
     }
-    background-color: var(--color-sidebar-surface);
+    /* Chrome level; the aspect derivation for this subtree keys off --surface-bg (see surface.css). */
+    --surface-bg: var(--dx-surface-chrome);
+    background-color: var(--surface-bg);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)